Assets/MixedRealityToolkit.Examples/Demos/StandardShader/Textures/Lander_albedo.png.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: 11056bb52f91cdc43ba4fba9accfb226
```

Comment:
How strongly do you feel about adding the lander model? The problem with adding these FBX assets into Examples it they bloat the UnityPackage file. The textures and mesh data adds like 7-8MB alone in this change. I know that is typical for assets but that's a 15% increase to the existing package. 

I recommended removing because of the size and primarily because it's the feature is not dependent on it. We can still demonstrate the feature with the moon lander rover or other existing FBX

Reply:
I would really like to add this model since it demonstrates proper asset consturction, naming, and PBR workflow. It would be nice to phase out the old lander (and maybe even the rover) since they don't demonstrate the same quality bar I would like to see in the MRTK.